server.js

```diff
@@ -13,52 +13,50 @@ const dev = process.env.NODE_ENV !== 'production';
 const nextApp = next({ dev });
 const nextHandler = nextApp.getRequestHandler();
 
-const listenedGame = {};
 const gameListeners = {};
 
-const setIntervalForListenedGame = (socket, idGame) => {
-    if (!listenedGame[idGame]) {
-        listenedGame[idGame] = setInterval(async () => {
-            let game = await getGame(idGame);
-            if (gameListeners[idGame])
-                gameListeners[idGame].forEach(s => {
-                    s.emit(`game${idGame}`, game);
-                });
-        }, 3000);
-        gameListeners[idGame] = [];
+const refreshGameForOpenedSockets = async idGame => {
+    if (gameListeners[idGame]) {
+        const game = await getGame(idGame);
+        gameListeners[idGame].forEach(s => {
+            s.emit(`game${idGame}`, game);
+        });
     }
+};
+
+const registerGameListener = (socket, idGame) => {
```

Comment:
refactor to gameListener exposing gamelistener array and returning register and unregister function, this will be easier to test